files/kpi/nginx.conf.tpl

```diff
@@ -8,10 +8,25 @@ server {
     client_max_body_size 100M;
     large_client_header_buffers 8 16k;
 
+    {{ if .Values.kpi.nginx.large_headers.enabled -}}
+    # Proxy buffer settings to handle large headers
+    proxy_buffer_size {{ .Values.kpi.nginx.large_headers.proxy_buffer_size | default "32k" }};
```

Comment:
Preference would be to not set defaults here but in the values.yaml file, so that the values.yaml file has a listing of all possible keys.

Reply:
I did that, but, based on other examples, I also left defaults in the nginx templ. 
Let me know if you want to remove them altogether.